packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -353,3 +353,8 @@ const vpnConnection = vpc.addVpnConnection('Dynamic', {
 });
 const state = vpnConnection.metricTunnelState();
 ```
+
+### VPC endpoints
+VPC gateway and interface endpoints can be added to a VPC:
```

Comment:
Copy & paste some official docs that explain what VPC endpoints are and why would I want to use them